File: Plugins/Aspose_Diagram_Java_for_Ruby/lib/asposediagramjava/Pages/getpageinfo.rb
module Asposediagramjava
  module GetPageInfo
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'

        # Call the diagram constructor to load diagram from a VSD file
        diagram = Rjb::import('com.aspose.diagram.Diagram').new(data_dir + "Drawing.vsd")

        #page = diagram.getPages().getPage(page_id)
        page = diagram.getPages().getPage(0)
        
        puts "Page ID : " + page.getName().to_s
    end
  end
end

File: Plugins/Aspose_Diagram_Java_for_PHP/src/aspose/diagram/WorkingwithPages/GetPageInfo.php
<?php
namespace Aspose\Diagram\WorkingwithPages;

use com\aspose\diagram\Diagram as Diagram;
use com\aspose\diagram\SaveFileFormat as SaveFileFormat;
use com\aspose\diagram\Page as Page;
class GetPageInfo{

    public static function run($dataDir=null){

        # Call the diagram constructor to load diagram from a VSD file
        $diagram = new Diagram($dataDir."drawing.vsd");

        #page = diagram.getPages().getPage(page_id)
        $page = $diagram->getPages()->getPage(0);

        print "Page ID : ".$page->getName().PHP_EOL;
    }
}